clickhouse-http-client/src/main/java/com/clickhouse/client/http/ClickHouseHttpConnection.java

```diff
@@ -59,6 +52,30 @@ public abstract class ClickHouseHttpConnection implements AutoCloseable {
     private static final byte[] SUFFIX_STRUCTURE = "_structure\"\r\n\r\n".getBytes(StandardCharsets.US_ASCII);
     private static final byte[] SUFFIX_FILENAME = "\"; filename=\"".getBytes(StandardCharsets.US_ASCII);
 
+    private static final String LOCAL_ADDRESS;
+
+    static {
+        // get local address but not localhost
+        String address = "";
+        try {
+            Enumeration<NetworkInterface> networkInterfaces = NetworkInterface.getNetworkInterfaces();
```

Comment:
Good day, @JackyWoo!
Working with OS in the static block would lead to problem on application startup and we need to avoid it. 
Another issue with static blocks is they may not be bypassed when not needed. 
Current change overrides default behavior for all client what may break someone's application if they rely on it.

Reply:
Thanks